.env.local.example

```
@@ -5,6 +5,8 @@ MOSQUE_API_ENDPOINT=http://localhost:3000/api/example-mosque
 THEME_COLOR_HIGHLIGHT=#29B0EA
 THEME_COLOR_PRIMARY=#003750
 THEME_COLOR_PRIMARY_ALT=#086182
+# Change local for date time in different locale
+LOCALE="en"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)